{"name":"Login Website","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertTrue(Assert.java:44)\n\tat org.testng.Assert.assertTrue(Assert.java:54)\n\tat tests.ProductStore.LoginTest.invalidUserNameLogin(LoginTest.java:45)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","description":"Testcase description : Verify login","steps":[],"attachments":[{"name":"Page screenshot","source":"68565577-2a1e-46fc-af4b-835a23135ad0-attachment","type":"image/png"},{"name":"invalidUserNameLogin failed and screenshot taken!","source":"bfa57e44-014c-4597-88f1-b9ce074ef51a-attachment","type":"text/plain"}],"parameters":[],"start":1606987462638,"stop":1606987464381,"uuid":"545a2b6f-f735-4d54-87b6-e7b3823b730a","historyId":"baa57fd5e303ffb4d4832aa49b56f941","fullName":"tests.ProductStore.LoginTest.invalidUserNameLogin","labels":[{"name":"package","value":"tests.ProductStore.LoginTest"},{"name":"testClass","value":"tests.ProductStore.LoginTest"},{"name":"testMethod","value":"invalidUserNameLogin"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium-framwork"},{"name":"subSuite","value":"tests.ProductStore.LoginTest"},{"name":"host","value":"BAEMINs-MacBook-Pro.local"},{"name":"thread","value":"9580@BAEMINs-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"blocker"}],"links":[]}